tests/conftest.py

```diff
@@ -421,3 +440,11 @@ def count_servers():
         # assert num_dpf_exe == 1
 
     request.addfinalizer(count_servers)
+
+
+# to call at the end
+core.server.shutdown_all_session_servers()
+try:
+    core.apply_server_context(core.AvailableServerContexts.premium)
+except core.errors.DPFServerException:
```

Comment:
@cbellot000 Can we not be a little bit more specific? To not catch any possible Server exception

Reply:
sorry it's a mistake